DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_code_split/README.md

```diff
@@ -0,0 +1,258 @@
+# Using the flows test 'device_code_split'
+
+This FPGA tutorial demonstrates the -fsycl-device-code-split=per_source|per_kernel
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
+| Hardware                          | Intel&reg; Programmable Acceleration Card (PAC) with Intel Arria&reg; 10 GX FPGA <br> Intel&reg; FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix&reg; 10 SX) <br> Intel&reg; FPGA 3rd party / custom platforms with oneAPI support <br> **Note**: Intel&reg; FPGA PAC hardware is only compatible with Ubuntu 18.04*
+| Software                          | Intel® oneAPI DPC++/C++ Compiler
+| What you will learn               | How to use the `-fsycl-device-code-split=per_source|per_kernel` to create multiple aocx.
+| Time to complete                  | 30 minutes
+
+> **Note**: Even though the Intel DPC++/C++ OneAPI compiler is enough to compile for emulation, generating reports and generating RTL, there are extra software requirements for the simulation flow and FPGA compiles.
+>
+> For using the simulator flow, one of the following simulators must be installed and accessible through your PATH:
+> - Questa*-Intel® FPGA Edition
+> - Questa*-Intel® FPGA Starter Edition
+> - ModelSim® SE
+>
+> When using the hardware compile flow, Intel® Quartus® Prime Pro Edition must be installed and accessible through your PATH.
+
+## Purpose
+The purpose of this tutorial is to describe the `-fsycl-device-code-split` flag and to show how it can be leveraged to break down the compilation of several kernels into smaller individual compiles.  
```

Comment:
I think we should clarify this a bit. 

This options splits and generates multiple FPGA bitstreams. The total compile time will likely be longer, but the main perk is if your image is getting too large to fit on the FPGA, this can be a nice way to split them.

Reply:
I think the "Purpose" section is just a one line description of what it does, the pro and cons can be added to the "Using the flag" section?

Reply:
Okay - that sounds good. Is this information already, or do we want to re-iterate it there?